music21/metadata/__init__.py

```diff
@@ -578,22 +567,19 @@ def copyright(self):
         >>> md = metadata.Metadata()
         >>> md.copyright is None
         True
-        >>> md['dcterms:rights'] = 'Copyright © 1984 All Rights Reserved'
+        >>> md['dcterms:rights'] = ('Copyright © 1984 All Rights Reserved',)
```

Comment:
outdated description of 'MULTIPLE' above.

The first example of using this property should be as simple as possible.  And again (I know the previous ones were poor), use examples that make musicians notice that this is for them, not just a general system:

```
>>> md.copyright = 'Copyright © 1896, Éditions Durand (expired)'
>>> md.copyright
'Copyright © 1896, Éditions Durand (expired)'
```

Then only after the simple case has been demonstrated, show adding them as `dcterms:rights` explaining what is happening there, etc.

Reply:
Done.

Reply:
Done.